### PR TITLE
Implement Frame.Context

### LINKED ISSUE #21: Implement Frame.Context tests
https://github.com/GoogleChrome/puppeteer/blob/v1.0.0/test/test.js#L602

### PR DESCRIPTION
**Pull request by kblok**
closes #21 

Based on #165. Let's wait for #165 to be merged before reviewing this guy

### CODE REVIEW

**Comment by Meir017** [comment]
the file `WaitForFunctionTests.cs` and `ContextTests.cs` got mixed up

File: lib/PuppeteerSharp/Frame.cs [hunk 1]
```diff
@@ -74,6 +74,10 @@ public Frame(Session client, Page page, Frame parentFrame, string frameId)
             return await context.EvaluateFunctionAsync<T>(script, args);
         }
 
+        /// <summary>
+        /// Gets the execution context associated with the frame.
```
Comment (Meir017) [inline lib/PuppeteerSharp/Frame.cs]: maybe use `<see cref="ExecutionContext"/>` instead of plane words?

**Comment by kblok** [comment]
@Meir017 ups

**Comment by kblok** [comment]
Done @Meir017

**Pull request merged**